bip-p2qrh.mediawiki

```diff
@@ -68,9 +68,11 @@ The proposal above also leaves a gap in case it makes sense to use version 2, or
 
 The name P2QRH means Pay to Quantum Resistant Hash. This is similar to how P2PKH, as defined in [https://github.com/bitcoin/bips/blob/master/bip-0016.mediawiki#specification BIP-16].
 
-P2QR addresses will be needed to maintain TapScript compatibility, and will encode the SQIsign public key. P2QRH addresses will use SHA-256 / MD160, similar to BIP-16, to reduce the size of new outputs.
+P2QR addresses will be needed to maintain TapScript compatibility, which will be defined in a separate BIP, and will encode the SQIsign public key. P2QRH addresses will use SHA-256 / MD160, similar to BIP-16, to reduce the size of new outputs.
```

Comment:
When look at this paragraph as a whole, I am not sure I understand this.  Are we trying to say that P2QR addresses are compatible with Tapscript?  Also, did you mean to say P2QRH and not P2QR in the first sentence?  If so, I would write something like this. 

"The proposed P2QRH address format is intended to be compatible with TapScript, which will be defined in a follow-on BIP.   Additionally, the address encodes the SQISign public key and uses SHA-256 / MD160 (similar to BIP-16) to reduce the size of the new outputs."

Note that I tried to use present tense where it made sense.

Reply:
No, P2QR is meant to be analogous to P2TR, in that it's TapScript compatible. P2QRH is not meant to support TapScript. I'll try to clarify that language.